src/main/java/com/shaneschulte/mc/clanarena/commands/CreateCmd.java

```diff
@@ -0,0 +1,379 @@
+package com.shaneschulte.mc.clanarena.commands;
+
+import com.shaneschulte.mc.clanarena.utils.CmdProperties;
+import com.shaneschulte.mc.clanarena.utils.MsgUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.boss.BossBar;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.player.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CreateCmd implements CmdProperties, Listener {
+
+    private static HashMap<String, BossBar> currentPlayerBossBars = new HashMap<>();
+    private static HashMap<String, String> currentArenaNameByPlayer = new HashMap<>();
+    private static int minSpawns = 6;
+
+    // Stage 01: selecting region
+    private static ArrayList<String> playersInTheMiddleOfDesignatingThe2PointsNeededToCreateAnArenaRegion = new ArrayList<>();
+
+    private static HashMap<String, Location> currentPos1ByPlayer = new HashMap<>();
+    private static HashMap<String, Location> currentPos2ByPlayer = new HashMap<>();
+
+    // Stage 02: selecting spawn points
+    private static ArrayList<String> playersInTheMiddleOfSelectingSpawnPoints = new ArrayList<>();
```

Comment:
This setup goes against Java's OOPiness; instead of having 1309253402 hash maps, you should have 1 hash map that points player UUIDs -> Arena, or something like that